arviz/plots/backends/matplotlib/lmplot.py

```diff
@@ -0,0 +1,44 @@
+"""Matplotlib plot linear regression figure."""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from ...hdiplot import plot_hdi
+
+
+def plot_lm(x, y, y_model, y_ppc, num_pp_samples, kind_pp, kind_model):
+    """Matplotlib Linear Regression."""
+    _, ax = plt.subplots(1, 1, figsize=(6, 4), dpi=100)
+
+    # plot data
+    ax.plot(x, y, marker=".", color="C3", lw=0, zorder=10)
```

Comment:
At some point these could be used with kwargs (with some setdefault magic)

Reply:
Different kwargs for observed data, PPC and model?